content/docs/reference-react.md

```diff
@@ -98,57 +98,57 @@ class Greeting extends React.Component {
 }
 ```
 
-See the [React.Component API Reference](/docs/react-component.html) for a list of methods and properties related to the base `React.Component` class.
+Veja a [referência da *API* para React.Component](/docs/react-component.html) para uma lista de métodos e propriedades relacionadas à classe base `React.Component`.
```

Comment:
```suggestion
Veja a [referência da *API* para o React.Component](/docs/react-component.html) para ver uma lista de métodos e propriedades relacionadas à classe base `React.Component`.
```

Reply:
I think that the verb "ver" is repeating a little bit too much :stuck_out_tongue: 

> **Veja** a referência para **ver**

What do you think about

> ~Veja~ **Consulte** a [referência da *API* para o React.Component](/docs/react-component.html) para ver uma lista de métodos e propriedades relacionadas à classe base `React.Component`.

Reply:
yup agree